### PR TITLE
Chore: Added tooltip to options of message-actions

### LINKED ISSUE #24042: Tooltip missing for options of message-actions
### Description:
Tooltip missing for options of message-actions

### Steps to reproduce:

1. Hover on any message. The message actions will appear.
2. Hover on the more options menu in the message actions.

### Expected behavior:
Tool tip should be displayed

### Actual behavior:
Nothing is displayed

https://user-images.githubusercontent.com/76481696/147605130-0aa2ed3c-04eb-4688-a231-d58f763e1dfd.mp4

Screenshot from 2021-12-29 02-05-51

### Server Setup Information:

- Version of Rocket.Chat Server: develop
- Operating System: Ubuntu Linux

### Client Setup Information

- Desktop App or Browser Version: Browser
- Operating System: Ubuntu Linux

### PR DESCRIPTION
**Pull request by aswinidev**
Fixes #24042

## Before

https://user-images.githubusercontent.com/76481696/147604643-253d0577-65d6-4bfe-b67e-49a7cc67c03e.mp4

## After

https://user-images.githubusercontent.com/76481696/147604656-9b2aa16b-7b90-40d2-bb29-b75bad3d5f15.mp4

Screenshot from 2021-12-29 01-59-06

## Issue(s)

Fixes #24042

### CODE REVIEW

File: app/ui-message/client/message.html [hunk 1]
```diff
@@ -141,7 +141,7 @@
 							</button>
 						{{/each}}
 					</div>
-					<div class="message-actions__menu" aria-haspopup="true">
+					<div class="message-actions__menu" aria-haspopup="true" title="Options">
```
Comment (ggazzo) [inline app/ui-message/client/message.html:144]: - Please use translation key here
- I think "more" is more suitable for this case

**Review by ggazzo — CHANGES_REQUESTED** [review]
There is no need to create an issue just to start to work on the PR, if you are willing to fix something, feel free to just create the PR

File: app/ui-message/client/message.html (hunk 1 above)
Comment (aswinidev) [reply app/ui-message/client/message.html:144]: I was thinking about translating but couldn't figure out how to translate here. Could you give me some hints? 
I saw {useTranslation} is being used in some files but i wasn't able to use it here

Comment (aswinidev) [reply app/ui-message/client/message.html:144]: I figured it out. Thanks for the review!

**Comment by aswinidev** [comment]
@ggazzo Please review

**Review by ggazzo — APPROVED** [review]

**Pull request merged**